CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ directly impact users rather than highlighting other key architectural updates.*
 - The Balances screen now provides details on current balances like Change pending and Pending transactions
 - The screen also adds a new Block synchronization progress bar and status, which were initially part of the Account 
   screen and redesigned
+- And finally, there is a new Transparent funds shielding panel
```

Comment:
Not sure if we want to comment changes this way (by starting and finally..) up to somebody else to decide but I would just informally state the fact. `A new transparent funds shielding panel added to the Balance tab`